### PR TITLE
qprint 1.1

### PR DESCRIPTION
**Pull request by bfontaine**

### CODE REVIEW

**Comment by DomT4** [comment]
It's having issues on >10.8.

```
==> brew uses qprint							Error: /usr/local/Library/Formula/qprint.rb:15: invalid multibyte char (US-ASCII)
/usr/local/Library/Formula/qprint.rb:15: invalid multibyte char (US-ASCII)
/usr/local/Library/Formula/qprint.rb:15: syntax error, unexpected end-of-input, expecting keyword_end
    msg = "tést hömêbrèw"
              ^
 FAILED
Error: /usr/local/Library/Formula/qprint.rb:15: invalid multibyte char (US-ASCII)
/usr/local/Library/Formula/qprint.rb:15: invalid multibyte char (US-ASCII)
/usr/local/Library/Formula/qprint.rb:15: syntax error, unexpected end-of-input, expecting keyword_end
    msg = "tést hömêbrèw"
              ^
```

Try making `# encoding: UTF-8` the first line of the formula.

**Comment by bfontaine** [comment]
Thanks @DomT4, strangely it worked locally. I edited the test to use the `\u` notation, let’s see how it goes.

File: Library/Formula/qprint.rb [hunk 1]
```diff
@@ -1,14 +1,19 @@
-require 'formula'
-
 class Qprint < Formula
-  homepage 'http://www.fourmilab.ch/webtools/qprint'
-  url 'http://www.fourmilab.ch/webtools/qprint/qprint-1.0.tar.gz'
-  sha1 '533a4942e93cccc2e6b3fd2171707bf1d0054d20'
+  homepage "http://www.fourmilab.ch/webtools/qprint"
```
Comment (DomT4) [inline Library/Formula/qprint.rb]: Can you shunt this and the `url` to `https://` please? Upstream seem to support it. Thanks!

File: Library/Formula/qprint.rb [hunk 2]
```diff
@@ -1,14 +1,19 @@
...
-  homepage 'http://www.fourmilab.ch/webtools/qprint'
-  url 'http://www.fourmilab.ch/webtools/qprint/qprint-1.0.tar.gz'
-  sha1 '533a4942e93cccc2e6b3fd2171707bf1d0054d20'
+  homepage "https://www.fourmilab.ch/webtools/qprint"
+  url "https://www.fourmilab.ch/webtools/qprint/qprint-1.1.tar.gz"
+  sha256 "ffa9ca1d51c871fb3b56a4bf0165418348cf080f01ff7e59cd04511b9665019c"
 
   def install
-    system "./configure", "--disable-debug", "--disable-dependency-tracking"
+    system "./configure", "--prefix=#{prefix}"
```
Comment (xu-cheng) [inline Library/Formula/qprint.rb]: The configure doesn't accept `--disable-debug` and `--disable-dependency-tracking`?

Comment (bfontaine) [inline Library/Formula/qprint.rb]: Nope, it gave me warnings:

```
configure: WARNING: unrecognized options: --disable-debug, --disable-dependency-tracking
```

Comment (xu-cheng) [inline Library/Formula/qprint.rb]: :+1:

**Pull request closed without merging**